### ISSUE TITLE
Consider adding a empty implementation for non-darwin platforms

### ISSUE THREAD

**Issue by gerges-zz** [ccnokes/electron-named-image#3]
To prevent the use of optional dependencies and try/catch wrapping `requrie`, have the module define a blank implementation of the api for unsupported platforms.

**Comment by ccnokes** [comment]
That's a good idea 👍 . I'll be able to get to it sometime this month (hopefully sooner). PRs are welcome too, of course 😸

**Comment by ccnokes** [comment]
Ok v1.0.3 fixes this.

**Comment by gerges** [comment]
Sorry I wasn't able to get to this, but thanks so much for taking care of it!

**Comment by ccnokes** [comment]
Ok I'll look into it. That error is a common symptom of not rebuilding them properly for Electron (more instructions here: https://github.com/electron/electron/blob/master/docs/tutorial/using-native-node-modules.md). Maybe you've done that already, just thought I'd double check.

**Comment by gerges** [comment]
Yeah I'm curious why other modules are able to bubble up "these modules were built with the wrong version of node" and this one errors this way.

**Comment by ccnokes** [comment]
Ok I was able to repro your issue and I got it working for me. So 1.0.4 *should* fix it. 😸
